src/js/api/urls.js

```diff
@@ -28,6 +28,8 @@ export const PICKLIST_ITEMS_EXPORT = (id) => `${STOCK_MOVEMENT_API}/exportPickLi
 export const PICKLIST_TEMPLATE_EXPORT = (id) => `${STOCK_MOVEMENT_API}/picklistTemplate/${id}`;
 export const PICKLIST_IMPORT = (id) => `${STOCK_MOVEMENT_API}/importPickListItems/${id}`;
 export const PACKING_LIST_TEMPLATE = `${STOCK_MOVEMENT_API}/packingList/template`;
+export const STOCK_MOVEMENT_IMPORT_CSV = (id) => `/stockMovement/importCsv/${id}`;
+export const STOCK_MOVEMENT_EXPORT_CSV = (id) => `/stockMovement/exportCsv/${id}`;
```

Comment:
If the URL is connected to the app view or other frontend thing, we should use `applicationUrl.js`
This file + services should be used for API calls.